### ISSUE TITLE
Fix Nutmeg image builds

### ISSUE THREAD

**Issue by jmbowman** [openedx-unsupported/devstack#965]
The devstack Docker image builder jobs for Nutmeg have been (belatedly) created, and are being triggered correctly, but many of them are still failing so we don't yet have a useful set of Nutmeg devstack images.  Please review the jobs at https://tools-edx-jenkins.edx.org/job/DockerCI/job/Nutmeg/job/image-builders/ and get them passing.  Feel free to create sub-issues as appropriate to address different problems you find, and when done please verify that the images have in fact been uploaded to https://hub.docker.com/u/edxops (check for nutmeg.master tags).

**Sub-tasks:**
- [x] https://github.com/openedx/devstack/issues/969
- [x] https://github.com/openedx/devstack/issues/970
- [x] https://github.com/openedx/devstack/issues/971

**Comment by aht007** [comment]
The jobs were failing due to a bug in `focal_common` image that is used as base image for all other images. I had a discussion with @nadeemshahzad from SRE and we temporarily fixed the issue. I have also created an [SRE ticket](https://2u-internal.atlassian.net/servicedesk/customer/portal/15/DOS-3446?created=true)(private to 2u org) for permanently fixing the `focal_common` image. Most of the jobs are running now and the images have been pushed to `edxops`. Some jobs are still failing for which I have created sub-tasks and the team is actively working on it.
